dockerfile/torch1.14-cuda11.8.dockerfile

```diff
@@ -55,3 +56,5 @@ RUN python3 -m pip install --upgrade pip && \
 ADD . .
 RUN python3 -m pip install . && \
     make postinstall
+
+ENV LD_PRELOAD="/usr/local/lib/libmsampfp8.so:/usr/local/lib/libnccl.so:${LD_PRELOAD}"
```

Comment:
libnccl.so is not necessary?
for libmsampfp8.so, what if user install the library and runs directly?

Reply:
For the 1st question: yes, libnccl.so is necessary because libmsampfp8 depends on libnccl. Withotu libnccl in LD_PRELOAD, we will get an error.
For the 2nd question: I add document to let user export LD_LIBRARY manually.